kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ProcessServiceIntegrationTest.java

```diff
@@ -236,7 +269,24 @@ public void testSignalProcessInstance() throws Exception {
             processClient.abortProcessInstance(CONTAINER_ID, processInstanceId);
             fail(e.getMessage());
         }
+    }
+    
+    @Test
+    public void testSignalProcessInstanceWrongContainer() throws Exception {
```

Comment:
Checked exceptions are never thrown in this test
```suggestion
    public void testSignalProcessInstanceWrongContainer() {
```